### DISCUSSION TITLE
Parameters for Imaginary Time Green's function

### DISCUSSION THREAD

**Discussion by AQuantumGuy** [TRIQS/triqs#927] [Q&A]
Dear TRIQS community,
I wanted to request some informations.

For testing, I've performed DFT+DMFT simulation with Quantum Espresso, Wannier90 and TRIQS as given from the tutorial on TRIQS repo (https://github.com/TRIQS/tutorials/blob/unstable/AbinitioDMFT/solutions/01s-solid_dmft.ipynb)

Now I'm working with the calculations of imaginary time Green's function on a quantum simulator. I'm using a private package for it.

As inputs for the calculations, the following parameters are needed (from DFT+DMFT simulation)

Hamiltonian parameters:
bath energy level
hybridization
U
chemical potential 

Temperature
Mesh of imaginary time

I think U, chemical potential and temperature (beta) are given from. Could you kindly let me know the steps to have those parameters? I don't have much experience with DFT and DMFT simulations, I've mostly followed the tutorials on triqs website like the one in the link provided. If you could give me info based on the example given that would be easier for me.

I appreciate your help!

**Comment by the-hampel** [comment]
Hi @AQuantumGuy ,

I am not sure I understand the question correctly. In the tutorial you are mentioning these parameters are discussed, see https://github.com/TRIQS/tutorials/blob/unstable/AbinitioDMFT/dmft_config.ini :
```
beta = 10
U = 3.6 
n_tau = 5001
```
The chemical potential of the converged DMFT calculation with the provided wannier Hamiltonian can be, as it is discussed, read from https://github.com/TRIQS/tutorials/blob/unstable/AbinitioDMFT/solutions/b10-U3.6/observables_imp0.dat:
```
 it |         mu | G(beta/2) per orbital |  orbital occs up+down |      impurity occ
...
  7 |   12.91505 |              -0.10431 |               0.99636 |           0.99636
  8 |   12.92424 |              -0.10370 |               1.00087 |           1.00087
  9 |   12.92160 |              -0.10397 |               1.00069 |           1.00069
 ```
So the chemical potential for the converged DMFT loop for this U value is ~12.9216. Your bath energy levels and hybridization have to be extracted from fitting the continuous hybridization function. cthyb does not use a discretized hyb function. You can use for that for example the converged `delta_tau` stored in the h5 archive and use https://triqs.github.io/triqs/unstable/documentation/python_api/triqs.gf.tools.discretize_bath.html#triqs-gf-tools-discretize-bath. Fitting the bath with a finite number of bath sites needs some training and some testig. 

Best,
Alex
